pkg/model/bootstrapscript.go

```diff
@@ -65,6 +67,8 @@ type BootstrapScript struct {
 
 	// nodeupConfig contains the nodeup config.
 	nodeupConfig fi.CloudupTaskDependentResource
+	// nodeupScript contains the nodeup boostrap script.
```

Comment:
Nit: typo in "bootstrap" and I would say "for use by karpenter" (and maybe a comment that we might not need it if we moved this workflow to toolbox - I think that's right?)

Reply:
Done